auth/role_manager.hh

```diff
@@ -76,6 +76,7 @@ public:
 };
 
 using role_set = std::unordered_set<sstring>;
+using direct_roles_map = std::multimap<sstring, sstring>;
```

Comment:
I don't think this name is descriptive enough. "Direct roles map" does not tell much about the relationship between keys and value.

How about `role_to_direct_grantees_map`? I don't like it too much either because it requires non-zero concentration in order to parse it, but I think it does describe the relationship correctly.

Reply:
`grantees` isn't a good name because (role1, role2) entry in the map means permissions of role2 are subset of role1's premissions.
Changed it to `role_to_direct_granted_map`.